code/modules/library/skill_learning/skillchip.dm

```diff
@@ -602,3 +602,174 @@
 
 	. += span_notice("<i>On the back of the chip, you see a list of songs:</i>")
 	. += span_smallnotice("<i>[jointext(songs, "<br>")]</i>")
+
+/obj/item/skillchip/acrobatics
+	name = "old F058UR7 skillchip"
+	desc = "A formerly cutting-edge skillchip that granted the user an advanced, Olympian-level degree of kinesthesics for flipping, spinning, and absolutely nothing else. \
+		It was pulled off the markets shortly after release due to users damaging the chip's integrity from excessive acrobatics, causing deadly malfunctions. It really puts the 'flop' in 'Fosbury Flop'!"
+	skill_name = "Spinesthetics"
+	skill_description = "Allows you to flip and spin at an illegal and dangerous rate."
+	skill_icon = FA_ICON_WHEELCHAIR_ALT
+	activate_message = span_notice("You suddenly have an extremely advanced and complex sense of how to spin and flip with grace.")
+	deactivate_message = span_notice("Your divine grasp of Spinesthesics disappears entirely.")
+	custom_premium_price = PAYCHECK_CREW * 4
+	// set integrity to 1 when mapping for !!FUN!!
+	max_integrity = 100
+	// more fun
+	var/list/affected_emotes = list("spin", "flip")
+	var/datum/effect_system/spark_spread/sparks
+	// you can use this without lowering integrity! let's be honest. nobody's doing that
+	var/allowed_usage = 3
+	var/reload_charge = 10 SECONDS
+	// current particle effect used for smoking brain
+	var/obj/effect/abstract/particle_holder/particle_effect
+
+/obj/item/skillchip/acrobatics/on_activate(mob/living/carbon/user, silent = FALSE)
+	. = ..()
+	RegisterSignal(user, COMSIG_MOB_EMOTE_COOLDOWN_CHECK, PROC_REF(whowee))
+
+/obj/item/skillchip/acrobatics/on_deactivate(mob/living/carbon/user, silent)
+	. = ..()
+	UnregisterSignal(user, COMSIG_MOB_EMOTE_COOLDOWN_CHECK)
+
+/obj/item/skillchip/acrobatics/Destroy(force)
+	qdel(sparks)
+	qdel(particle_effect)
+	. = ..()
+
+/obj/item/skillchip/acrobatics/proc/whowee(mob/living/carbon/bozo, emote_key, emote_intentional)
+	SIGNAL_HANDLER
+
+	if(!(emote_key in affected_emotes))
+		return
+
+	if(allowed_usage)
+		allowed_usage--
+		addtimer(CALLBACK(src, PROC_REF(charge)), reload_charge)
+	else
+		take_damage(1, sound_effect = FALSE)
+
+	if(!sparks)
+		sparks = new(src)
+
+	// minimum roll is by default capped at 50, with the min value lowering as integrity is reduced.
+	var/mintegrity = clamp(50 - (100 - get_integrity()), 1, 100)
+	switch(rand(mintegrity, get_integrity())) // 1 to 100 but gets worse every time
+		// CRIT FAIL
+		if(1)
+			bozo.visible_message(span_userdanger("[bozo]'s head suddenly explodes outwards!"))
+
+			explosion(bozo, light_impact_range = 2, adminlog = TRUE, explosion_cause = src)
+			// WITNESS THE GORE
+			for(var/mob/living/splashed in view(2, bozo))
+				if(bozo.has_status_effect(/datum/status_effect/grouped/blindness))
+					to_chat(splashed, span_userdanger("You're covered in blood!"))
```

Comment:
```suggestion
					to_chat(splashed, span_userdanger("You're splashed with something!"))
```
If you're blind, how can you tell it's exactly blood (except from the slight smell of iron)?

Reply:
anosmia content